components/webcompat/features.cc

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/webcompat/features.h"
+
+#include "base/feature_list.h"
+
+namespace webcompat {
+namespace features {
+
+BASE_FEATURE(kBraveWebcompatExceptionsService,
```

Comment:
comment

Reply:
also, what would happen if/when the feature is enabled and then disabled? will the browser ignore existing rules?

Reply:
Comment added.

After the feature is enabled and disabled, the browser will ignore all webcompat rules, including those that previously existed when the feature was enabled.